src/main/java/ch/bfh/bti7081/view/NewSeminarView.java

```diff
@@ -1,22 +1,183 @@
+    private FormLayout dateComposite = new FormLayout(seminarDate,seminarTime);
+
+    private TextArea seminarDescription = new TextArea("Beschreibung");
+    private TextField seminarLink = new TextField("Externer Link");
+
+    private TextField seminarStreet = new TextField("Strasse");
+    private TextField seminarStreetNbr = new TextField("Nr.");
+    private FormLayout streetComposite = new FormLayout(seminarStreet,seminarStreetNbr);
+
+    private NumberField seminarPlz = new NumberField("PLZ");
+    private TextField seminarPlace = new TextField ("Ort");
+    private FormLayout placeComposite = new FormLayout(seminarPlz, seminarPlace);
+
+    private FormLayout seminarForm = new FormLayout();
+
+    private Label errorMessage = new Label("");
+
+    private Button save = new Button("Save", new Icon(VaadinIcon.PLUS));
+    private Button cancel = new Button("Cancel", new Icon(VaadinIcon.EXIT));
+
+    private HorizontalLayout formActions = new HorizontalLayout(save,cancel);
+
+
+    private Binder<Seminar> binder = new Binder<>();
+    private Seminar newSeminar = new Seminar();
+
     public NewSeminarView(){
-        NewSeminarViewImpl view = new NewSeminarViewImpl();
-        NewSeminarPresenter presenter = new NewSeminarPresenter(view);
-        view.setPresenter(presenter);
+        seminarForm.add(seminarTitle);
+        seminarForm.add(seminarCategory);
 
+        seminarForm.add(dateComposite);
+        seminarForm.add(seminarLink);
+        seminarForm.add(streetComposite);
+        seminarForm.add(placeComposite);
 
-        add(view);
-    }
+        seminarForm.add(seminarDescription);
 
+        seminarTitle.setRequiredIndicatorVisible(true);
+        seminarCategory.setRequiredIndicatorVisible(true);
+        //dateTimeComposite
+        seminarDate.setRequiredIndicatorVisible(true);
+        seminarTime.setRequiredIndicatorVisible(true);
+        //streetComposite
+        seminarStreet.setRequiredIndicatorVisible(true);
+        seminarStreetNbr.setRequiredIndicatorVisible(true);
+        //placeComposite
+        seminarPlz.setRequiredIndicatorVisible(true);
+        seminarPlace.setRequiredIndicatorVisible(true);
+        seminarLink.setRequiredIndicatorVisible(true);
+        seminarDescription.setRequiredIndicatorVisible(true);
 
+        //Category-Settings
+        seminarCategory.setLabel("Kategorie");
+        seminarCategory.setItems(SeminarCategoryManager.getSeminarCategories());
+        seminarCategory.setTextRenderer(SeminarCategory::getName);
+        seminarCategory.setItemLabelGenerator(SeminarCategory::getName);
+        seminarCategory.setEmptySelectionAllowed(false);
 
+        //Binder-Configuration
+        binder.forField(seminarTitle).asRequired("Bitte Titel angeben").
+                withValidator(title -> title.trim().length() >= SeminarManager.minTitleLength, "Titel muss aus mind. "+SeminarManager.minTitleLength + " Zeichen bestehen").
```

Comment:
gemäss MVP-Pattern sollte die View nicht direkt mit dem SeminarManager zu tun haben

Reply:
geit da nume um die werte vor minimum längi, die chame guet ine angeri klass uslagere, i ha eifach nid so genau gwüsst wo wiuse sowou d klass seminar wie oh d view klass brucht. aber luegemer das doch morn ah :)

Reply:
Stimme in diesem Punkt nach der Diskussion mit dem Dozenten zu, m.E. muss der Aufruf an den Presenter gehen und der muss die Länge zurückgeben, o.ä.

Reply:
Umgesetzt in presenter und mit Enum (isch huere geil imfau bini richtig stouz)